src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx

```diff
@@ -0,0 +1,208 @@
+import { ViewMode } from '../../../common/Layout/ProjectsLayout/MobileProjectsLayout';
+import ImageSlider from './microComponents/ImageSlider';
+import { ExtendedProject } from '../../../common/types/projectv2';
+
+interface ProjectInfoSectionProps {
+  project: ExtendedProject;
+  isMobile: boolean;
+  setSelectedMode: SetState<ViewMode> | undefined;
+}
+
+const ProjectInfoSection = ({
+  project,
+  isMobile,
+  setSelectedMode,
+}: ProjectInfoSectionProps) => {
+  const tCountry = useTranslations('Country');
+  const {
+    metadata,
+    tpo,
+    videoUrl,
+    images,
+    website,
+    reviews,
+    description,
+    purpose,
+    expenses,
+    certificates,
+  } = project;
+  const isTreeProject = purpose === 'trees';
+  const isConservationProject = purpose === 'conservation';
+
+  const shouldRenderKeyInfo = useMemo(() => {
+    if (!isTreeProject && !isConservationProject) return false;
+    const generalConditions = [metadata.employeesCount];
+
+    const treeProjectConditions = isTreeProject
+      ? [
+          metadata.yearAbandoned,
+          metadata.firstTreePlanted,
+          metadata.plantingDensity,
+          metadata.maxPlantingDensity,
+          metadata.plantingSeasons && metadata.plantingSeasons.length > 0,
+          metadata.degradationYear,
+        ]
+      : [];
+
+    const conservationProjectConditions = isConservationProject
+      ? [
+          metadata.activitySeasons && metadata.activitySeasons.length > 0,
+          metadata.startingProtectionYear,
+        ]
+      : [];
+
+    return (
+      (isTreeProject && treeProjectConditions.some(Boolean)) ||
+      (isConservationProject && conservationProjectConditions.some(Boolean)) ||
+      generalConditions.some(Boolean)
+    );
+  }, [metadata, isTreeProject, isConservationProject]);
+
+  const shouldRenderAdditionalInfo = useMemo(() => {
+    const {
+      mainChallenge,
+      siteOwnerName,
+      longTermPlan,
+      acquisitionYear,
+      motivation,
+      mainInterventions,
+    } = metadata;
+    const generalConditions = [
+      mainChallenge,
+      siteOwnerName,
+      longTermPlan,
+      acquisitionYear,
+      motivation,
+      mainInterventions && mainInterventions?.length > 0,
+    ];
+    const treeProjectConditions = isTreeProject
+      ? [
+          metadata.siteOwnerType && metadata.siteOwnerType?.length > 0,
+          metadata.degradationCause,
+        ]
+      : [];
+    const conservationProjectConditions = isConservationProject
+      ? [
+          metadata.landOwnershipType && metadata.landOwnershipType?.length > 0,
+          metadata.actions,
+          metadata.socialBenefits,
+          metadata.ecologicalBenefits,
+          metadata.coBenefits,
+          metadata.benefits,
+          metadata.ownershipType,
+        ]
+      : [];
+
+    return (
+      generalConditions.some(Boolean) ||
+      treeProjectConditions.some(Boolean) ||
+      conservationProjectConditions.some(Boolean)
+    );
+  }, [metadata]);
```

Comment:
Add some comments about the logic for shouldRenderAdditionalInfo and shouldRenderKeyInfo as it could be confusing if I revisit this code later.